lib/authentication/view/sign_up_view.dart

```diff
@@ -45,25 +51,46 @@ class _SignUpViewState extends State<SignUpView> {
     _fetchFilter();
   }
 
+  /// This function takes the email of user and sends the name data to the name fields.
+  /// Special characters such as ".", "_" are used to separate the names, and numbers are removed.
+  /// Format example: firstnameone_firstnametwo.lastname123@stud.acs.pub.ro
+  /// The names will start with capital letters.
+
```

Comment:
https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

And statements like "this function does x" are redundant because this specific comment refers to the function anyway, so you can just say "do x".
```suggestion
  /// Attempt to guess the user's first and last name from the email, since university e-mail addresses are standardized.
  ///
  /// Special characters such as ".", "_" are used to separate the names, numbers are removed and names are capitalized.
  /// *Format example:* firstnameone_firstnametwo.lastname123@stud.acs.pub.ro
```